### DISCUSSION TITLE
Room 잠금 기능 사용 방법

### DISCUSSION THREAD

**Discussion by diddntjd99** [naverz/zepeto-studio-kor#65] [5. Q&A]
안녕하세요,
이번 업데이트로 room 인원 제한이나, 잠금 기능이 생겼다고 들었는데
room 잠금 기능 사용 예제를 알 수 있을까요?

**Comment by hyeonjin-jo** [comment]
안녕하세요

해당 기능 관련 문서가 곧 공개 예정입니다

공개가 된다면 추가 댓글 남겨드리겠습니다

**Comment by hyeonjin-jo** [comment]
https://github.com/naverz/zepeto-studio-kor/discussions/48

위의 답변 참고부탁드립니다
